File: LogAndroid/TrustarApp/src/main/kotlin/jp/co/toukei/log/trustar/chat/ChatRoomWithUsers2.kt
package jp.co.toukei.log.trustar.chat

import jp.co.toukei.log.trustar.db.chat.entity.ChatRoom
import jp.co.toukei.log.trustar.db.chat.entity.ChatUserWithReadRow

class ChatRoomWithUsers2(
    @JvmField val room: ChatRoom,
    @JvmField val users: List<ChatUserWithReadRow>,
)

File: LogiOS/TrustarApp/Chat/ChatRoomWithUsers2.swift
//
//  ChatRoomWithUsers2.swift
//  TrustarApp
//
//  Created by DionSoftware on 2022/11/25.
//  Copyright © 2022 DionSoftware. All rights reserved.
//

import Foundation

class ChatRoomWithUsers2 {
    var room: ChatRoom
    var users: [ChatUserWithReadRow]
    init(room: ChatRoom, users: [ChatUserWithReadRow]) {
        self.room = room
        self.users = users
    }
}
